README.md

```diff
@@ -1,3 +1,20 @@
-# Welcome!
+LibreClinica
+============
 
-LibreClinica is the community driven successor of OpenClinica. It is an open source clinical trial software for Electronic Data Capture (EDC) Clinical Data Management (CDM). https://libreclinica.org
+_the real open source electronic data capture (EDC) for clinical studies._
+
+[LibreClinica](https://libreclinica.org) is the community driven successor of OpenClinica. It is an open source clinical trial software for Electronic Data Capture (EDC) and Clinical Data Management (CDM).
+
+### Getting Started
+
+Check out [LibreClinica Documentation](https://libreclinica.org/documentation) to learn how to install, validate and use the software.
+
+### Development
+
+In case of interest in contributing into the codebase, check out [LibreClinica Developer Documentation](https://libreclinica-docs.readthedocs.io) to learn how to set up the development workstation and understand the branching strategy that is used to control the contribution workflows. 
+
+### Contributions
+                          
```

Comment:
What about adding a link to https://libreclinica.org/goals.html here?

Reply:
ok, I will add this as well